ios-sdk/src/MapViewController.swift

```diff
@@ -61,6 +93,11 @@ public class MapViewController: TGMapViewController, LocationManagerDelegate {
         if (shouldShowCurrentLocation) {
             markerSetVisible(marker, visible: true)
         }
+
+        if (shouldFollowCurrentLocation) {
+            print("Updating for current lat: \(location.coordinate.latitude) & long: \(location.coordinate.longitude)")
+            centerOnCurrentLocation()
+        }
     }
 
     public func authorizationDidSucceed() {
```

Comment:
Ok I see, requesting a location might be good, so as soon as you give the authorization the user see his current location from closer zoom level.